enterprise/backend/test/metabase_enterprise/serialization/v2/load_test.clj

```diff
@@ -1179,3 +1179,45 @@
                  (t2/select-one-fn select-target DashboardCard :entity_id (:entity_id dc2))))
           (is (= (:id dash1)
                  (t2/select-one-fn select-target DashboardCard :entity_id (:entity_id dc3)))))))))
+
+(deftest skip-errors-test
+  (let [change-ser    (fn [ser changes]
+                        (vec (for [entity ser
+                                   :let   [change (get changes (:entity_id entity))]]
+                               (if change
+                                 (merge entity change)
+                                 entity))))
+        logs-contain? (fn [msg-re logs]
+                        (some #(re-find msg-re %)
+                              (->> logs
+                                   (map (fn [[_log-level _error message]] message))
+                                   (into #{}))))]
+    (mt/with-empty-h2-app-db
+      (mt/with-temp [Collection coll {:name "coll"}
+                     Card       c1   {:name "card1" :collection_id (:id coll)}
+                     Card       c2   {:name "card2" :collection_id (:id coll)}
+                     Card       _c3  {:name "card3" :collection_id (:id coll)}]
+        (testing "It's possible to skip a few errors during extract"
+          (let [extract-one serdes/extract-one]
+            (with-redefs [serdes/extract-one (fn [model-name opts instance]
+                                               (if (= (:entity_id instance) (:entity_id c1))
+                                                 (throw (ex-info "Skip me" {}))
+                                                 (extract-one model-name opts instance)))]
+              (is (logs-contain? #"Skipping \w+ \d+ because of an error extracting it"
```

Comment:
It would be nice to test that its the expected entity (only) that's skipped. Not sure whether it's easier by building a literal regex or introspecting `errors`.